### ISSUE TITLE
Question: There a way to use RDP with SISH ?

### ISSUE THREAD

**Issue by metabsd** [antoniomika/sish#203]
Right now I'm using `ngrok tcp 3390` I change the default port of RDP on my laptop.

Ngrok generate a tcp connection to ngrok server and I can access RDP on my laptop.

MYLAPTOP -> NGROK <- accessing mylaptop with RDP Client using the url provide by ngrok.

I already setup sish locally, I successfully ssh forward tcp 3390 with ssh command.

```
ssh -R mylaptop:3390:localhost:3390 sish.mydomain.blah

Starting SSH Forwarding service for tcp:3390. Forwarded connections can be accessed via the following methods:
HTTP: http://....
HTTPS: https://...
```

Please help!!!
Thx in advance!

**Comment by antoniomika** [comment]
Hey @metabsd,

Yep is is possible! Just drop the `mylaptop` part of the ssh command to look like:

```bash
ssh -R 3390:localhost:3390 sish.mydomain.blah
```

sish will then just forward a raw TCP connection from sish.mydomain.blah:3390 -> localhost:3390. If you would rather this connection not be exposed to the open internet, you can use [alias forwarding](https://github.com/antoniomika/sish#tcp-alias-forwarding) which just exposes the connection internally to sish. Then, you can create a tunnel like the following:

```bash
ssh -R mylaptop:3390:localhost:3390 sish.mydomain.blah
```

And access the RDP service on another laptop by doing a local port forward to sish

```bash
ssh -L 3390:mylaptop:3390 sish.mydomain.blah
```

And connecting to `localhost:3390` via RDP. Please note that tcp aliases need to be explicitly enabled

I'll leave this open until you close it just in case you have more questions, you can also find me in #sish on Freenode or Libera

**Comment by metabsd** [comment]
Thx @antoniomika for helping me via Libera #sish

This is my docker-compose.yml file. 
I use port 2222 because I installed ssh on my VM so I have to use different port.
Also I used container tag `main` to get some other bug fix from @antoniomika 

```
version: '3.7'

services:
  sish:
    image: antoniomika/sish:main
    container_name: sish
    volumes:
      - ./keys:/keys
    command: |
      --ssh-address=:2222
      --private-key-location=/keys/ssh_key
      --bind-random-aliases=false
      --bind-random-ports=false
      --bind-random-subdomains=false
      --domain=sish.mydomain.blah
      --tcp-aliases=true
      --idle-connection=false
      --private-key-passphrase=supersecretpassphrase
    network_mode: host
    restart: always
```

My use case is to be able to use RDP Client. Destination == the RDP server, Source == RDP Client side.

Destination: `ssh -R mylaptop:3390:localhost:3390 sish.domain.blah -p 2222`

Where `mylaptop` is the alias, `3390` is the port I use for RDP Server(I changed the default one) and `-p 2222` is the port my container expose for ssh.

Source: `ssh -L 3390:mylaptop:3390 sish.domain.blah -p 2222`

At the end I use RDP Client and connect via localhost:3390.

I don't have to open anything on internet and I don't need a public real DNS. I use dnsmasq in my local network.

This is awesome from @antoniomika . Great Project! Great work!

Thx!!!
